src/main/java/com/microsoft/azure/relay/ClientWebSocket.java

```diff
@@ -156,7 +156,7 @@ boolean isOpen() {
 	 * 
 	 * @return Returns a CompletableFuture which completes when websocket receives text messages
 	 */
-	CompletableFuture<String> readTextAsync() {
+	public CompletableFuture<String> readTextAsync() {
```

Comment:
>public C [](http://example.com/codeflow?start=1&length=8)

Why do you need to make the two methods in this class public? `WebSocketChannel` is more public facing and `ClientWebSocket` is more of an internal implementation class, so I would prefer if you leave these methods internal.

Reply:
You can expose public methods in `WebSocketChannel` class instead.